tests/conftest.py

```diff
@@ -2,19 +2,15 @@
 #
 # This file is part of Invenio.
 # Copyright (C) 2016-2019 CERN.
+# Copyright (C) 2025 Northwestern University.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 
 """Pytest configuration."""
 
-from __future__ import absolute_import, print_function
-
-import imp
```

Comment:
Deprecated and removed package in modern Pythons . 

Removed the fixtures using it: not sure the tests were actually verifying anything they were doing in the first place.